db/migrate/20250917185540_add_address_record_to_ownerships.rb

```diff
@@ -0,0 +1,7 @@
+class AddAddressRecordToOwnerships < ActiveRecord::Migration[8.0]
+  def change
+    add_reference :ownerships, :address_record, index: true
+    add_reference :bikes, :address_record, index: true
```

Comment:
NOTE: adding address_record to bike here (rather than in #2911) so that the migration happens before the logic is added to prevent attribute out of date issues when deploying the update.